### ISSUE TITLE
CRT: add token price charts

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/atlas#6212]
## Context
Based on creators feedback it was expressed that purchasing tokens based on price time series would help with conversion. 

## Scope
Introduce token price charts to Gleev showing historical price movements as line chart.

**Comment by bedeho** [comment]
But where? and for who, I think this needs more of a user need focused description from them.  Also realise the price chart will for many look totally boring, as trading frequency is going to be extremely low.

**Comment by mochet** [comment]
I think also the same chart could also show historic channel revenue to better inform users of the "trend" / future value propositions of a channel. i.e. if a channel has 3k $JOY revenue to date, how long did it take to acquire that?

> Also realise the price chart will for many look totally boring, as trading frequency is going to be extremely low.
I have already seen some token sells happening in the block explorer, but its hard to tell where they are happening without access to charts--I think this is a case where this feature is needed to have more frequent trading activity.

**Comment by dmtrjsg** [comment]
@bedeho for tokens traded based on the bonding curve, I find this view pretty good. It has also the transactions view in a separate tab using external library shared by many token trading platforms, so we could look into smth like that:

https://mint.club/token/degen/DEPE

**Comment by bedeho** [comment]
Yeah, that shows the curve itself, I dont think that relates to the problem you describe. My interpretation is that creators are saying tht seeing a price history is an important part of buying a tradeable asset. Im not totally sure what people would be looking for, perhaps that there is a trend, but the bonding curve and current point on it is quite a static view that does not communicate dynamics well.

**Comment by msmadeline** [comment]
@dmtrjsg Hey I've added the price chart. Let me know wdyt

➡ https://www.figma.com/design/POw4j95jszfIP5bGfas2IC/Channel?node-id=3946-27116&t=LhLKslJUjlZWKrkb-4

**Comment by msmadeline** [comment]
@dmtrjsg Hey, I proposed some other placements for the price chart. Check it out

➡https://www.figma.com/design/POw4j95jszfIP5bGfas2IC/Channel?node-id=3956-25671&t=V1Ieqgpui69WPnQu-4

**Comment by bedeho** [comment]
Given that there are no controls to select the candle size, you need to define what it should be, although I would suggest adding basic controls.  Also, what time window will be shown? All time chart? last x days, weeks? Here controls could be of use also, but default is also needed.

**Comment by dmtrjsg** [comment]
@msmadeline I think based on the fact that most descriptions are empty best to place it between benefits and Description to start with. 

@bedeho I assumed this comes as part of the library itself, just mimicking what Dextools has for instance. 
In terms of defaults last 60 days with 1day candle would be best to start with imo. 

Based on my conversastion with @ikprk the bulk of the work would be on Orion side, and if we get 15m candle granularity this should be fine. 

Toggle for candle size could be: 15m, 1h, 1d, 1w. This may come hand in hand with the implementation of Discovery page price increments update and minimum window can be inherited from there..
